src/components/popup-menu/popup-menu-list.jsx

```diff
@@ -69,14 +69,24 @@ class PopupMenuList extends Component {
   };
 
   render() {
-    const { width, classes, children, enter, exit, maxHeight, isOpen, 'aria-labelledby': ariaLabelledBy } = this.props;
+    const {
+      width,
+      classes,
+      children,
+      enter,
+      exit,
+      maxHeight,
+      isOpen,
+      'aria-labelledby': ariaLabelledBy,
+      popupMenuDropdownZIndex,
```

Comment:
Either way I would prefer to have this named `zIndex`

Reply:
I prefer to have it explicit what I'm setting the z-index of

Reply:
what about `dropdownZIndex` then?